tests/testthat/indention_curly_brackets/multi_line_curly_round_only-in_tree

```diff
@@ -0,0 +1,39 @@
+ROOT (token: short_text [newlines/spaces])            
+ °--expr:  [0/0]                                      
+     ¦--expr:  [0/1]                                  
+     ¦   °--SYMBOL: a [0/0]                           
+     ¦--LEFT_ASSIGN: <- [0/1]                         
+     °--expr:  [0/0]                                  
+         ¦--FUNCTION: funct [0/0]                     
+         ¦--'(': ( [0/0]                              
+         ¦--SYMBOL_FORMALS: x [0/0]                   
+         ¦--')': ) [0/1]                              
+         °--expr:  [0/0]                              
+             ¦--'{': { [1/0]                          
+             ¦--expr:  [2/0]                          
+             ¦   ¦--expr:  [0/1]                      
+             ¦   ¦   °--SYMBOL: x [0/0]               
+             ¦   ¦--LEFT_ASSIGN: <- [0/1]             
+             ¦   °--expr:  [0/0]                      
+             ¦       ¦--expr:  [0/0]                  
+             ¦       ¦   ¦--expr:  [0/0]              
+             ¦       ¦   ¦   °--SYMBOL: pi [0/0]      
+             ¦       ¦   ¦--'(': ( [0/0]              
+             ¦       ¦   °--')': ) [0/0]              
+             ¦       ¦--'(': ( [0/0]                  
+             ¦       ¦--',': , [1/7]                  
+             ¦       ¦--',': , [1/0]                  
+             ¦       °--')': ) [0/0]                  
+             ¦--expr:  [1/8]                          
+             ¦   ¦--IF: if [0/0]                      
+             ¦   ¦--'(': ( [0/0]                      
+             ¦   ¦--expr:  [0/0]                      
+             ¦   ¦   ¦--'{': { [1/4]                  
+             ¦   ¦   ¦--expr:  [1/16]                 
+             ¦   ¦   ¦   ¦--expr:  [0/0]              
+"             ¦   ¦   ¦   ¦   °--STR_CONST: ""done [0/0]"
```

Comment:
It's a double quote escaping issue. I fixed it using `write.table()` which allows for more options than `write_delim()`, so it should be fine now.